main.py

```
@@ -655,7 +648,7 @@ def _is_zssm_trigger(text: str) -> bool:
             return False
         t = text.strip()
         # 忽略常见前缀：/ ! ！ . 。 、 ， - 等，匹配起始处 zssm
-        if re.match(r"^[\s/!！。\.、，\-]*zssm(\s|$)", t, re.I):
+        if re.match(r"^[\s/!！。\.、，\-]*(zssm|hyw|何意味)(\s|$)", t, re.I):
```

Ruff (0.15.1)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)